### ISSUE TITLE
offline version of historical database

### ISSUE THREAD

**Issue by zcaudate** [ripple/rippled-historical-database#171]
Is there a way to get an offline version of the transactions besides using the web api?

**Comment by shekenahglory** [comment]
I'm not sure what you mean by an offline version - you mean downloading the transaction history?

**Comment by zcaudate** [comment]
yes. currently it's only possible to download through the web api interface. Is there something like a zipped version on s3?
